L2 nightly reduction job is failing on main

https://github.com/tenstorrent/tt-metal/actions/runs/25540865150/job/74996944028#step:6:3135
```
FAILED tests/ttnn/nightly/unit_tests/operations/reduction/test_prod_nc.py::test_prod_dims[0-2, 3, TILE_HEIGHT * 6 - 1, TILE_WIDTH * 7 - 1] - RuntimeError: TT_FATAL @ /project/tt_metal/impl/tensor/tensor_apis.cpp:816: tensor.padded_shape()[-2] % constants::TILE_HEIGHT == 0 && tensor.padded_shape()[-1] % constants::TILE_WIDTH == 0
info:
Last 2 dims of input shape must be multiples of 32
```

Needs to be fixed

--- a/tests/ttnn/nightly/unit_tests/operations/reduction/test_prod_nc.py
+++ b/tests/ttnn/nightly/unit_tests/operations/reduction/test_prod_nc.py
@@ -73,9 +73,7 @@ def test_prod_dims(input_shape, dims, device):
     torch_output = torch.prod(torch_input, dims[0], True)
 
     cpu_layout = ttnn.ROW_MAJOR_LAYOUT
-    tt_output_cpu = (
-        ttnn.prod(tt_input, tt_output, dims=dims).cpu().to(cpu_layout).unpad_from_tile(output_shape).to_torch()
-    )
+    tt_output_cpu = ttnn.prod(tt_input, tt_output, dims=dims).cpu().to(cpu_layout).to_torch()
 
     rtol = atol = 0.1
     passing, output_pcc = comp_allclose_and_pcc(torch_output, tt_output_cpu, pcc=0.999, rtol=rtol, atol=atol)

